morango/models.py

```diff
@@ -191,16 +219,19 @@ class Store(AbstractStore):
     """
 
     id = UUIDField(primary_key=True)
-    # used to know which store records to deserialize into app layer
+    # used to know which store records need to be deserialized into the app layer models
     dirty_bit = models.BooleanField(default=False)
 
 
-class Buffer(AbstractStore):
+class Buffer(AbstractStore, UUIDModelMixin):
     """
-    ``Buffer`` is where records from the internal store are kept temporarily,
-    until they are sent or received by another morango instance.
+    ``Buffer`` is where records from the internal store are queued up temporarily, before being
+    sent to another morango instance, or stored while being received from another instance, before
+    dequeuing into the local store.
     """
 
+    uuid_input_fields = ("transfer_session_id", "model_uuid")
```

Comment:
Main reason was to avoid accidental duplication of buffer records if the same chunk was pushed/pulled twice.

Probably could achieve the same objective with `unique_together`? Any other ideas?